### PR TITLE
Converted some less util-functions depending apis to Python 2 and 3 compatible.

### PR DESCRIPTION
**Pull request by ShifaSZ**
Changes:
1. converted `dict.keys()/items()/values()` and `e.message` as usual
2. used `from future.standard_library import install_aliases` to solve the difference of `urllib` between Python 2 and 3.

### CODE REVIEW

File: seqr/views/apis/users_api.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
+from __future__ import unicode_literals
+from future.standard_library import install_aliases
+install_aliases()
```
Comment (hanars) [inline seqr/views/apis/users_api.py:3]: what does this do? If it is something we need, can you add a comment here explaining it?

File: seqr/views/apis/users_api.py [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
+import urllib.parse as urlparse
```
Comment (hanars) [inline seqr/views/apis/users_api.py:5]: I'm assuming this is a python 3 library support thing, but for my curiosity can you explain why this needs to update?

**Comment by hanars** [comment]
This looks awesome! I just have some clarifying questions I'd like to understand before merging which is why I didn't approve yet, but it looks really good. The other thing I'd suggest is we should periodically bump the minimum python 3 test coverage in travis.yml. Right now it is at 23% but it looks like with this PR we are over 50

File: seqr/views/apis/users_api.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/views/apis/users_api.py:3]: It installs the aliases for `urllib.parse` for Python 2.
In Python 3, some of the methods of `urllib` have been moved under `urllib.parse`. For example, `urllib.quote_plus()` becomes `urllib.parse.quote_plus()`. The `future` package provides an alias of `urllib.parse` to support Python 2&3 compatible code.

File: seqr/views/apis/users_api.py (hunk 2 above)
Comment (ShifaSZ) [reply seqr/views/apis/users_api.py:5]: Python 3 has `urllib.parse` while Python 2 doesn't. The code:
```
from future.standard_library import install_aliases
install_aliases()
```
makes Python 2 support `urllib.parse` as well.

File: seqr/views/apis/users_api.py (hunk 1 above)
Comment (ShifaSZ) [reply seqr/views/apis/users_api.py:3]: I will add comments to the source file.

**Pull request merged**